src/PKSim.Core/Commands/SetExpressionProfileUnitCommand.cs

```diff
@@ -5,32 +5,36 @@
 using PKSim.Core.Model;
 using PKSim.Core.Services;
 
-namespace PKSim.Core.Commands;
-
-public class SetExpressionProfileUnitCommand: SetParameterUnitCommand
+namespace PKSim.Core.Commands
 {
```

Comment:
Just a usual replacement of `namespace XXX;` with `namespace XXX{}` in order to be able to compile in VS2019 :)